proposals/tf_utf8_identifiers/README.md

```diff
@@ -0,0 +1,435 @@
+# Unicode Identifiers in USD
+
+Copyright &copy; 2022-2023, NVIDIA Corporation, version 1.1
+
+Aaron Luk  
+Matthew Kuruc  
+Edward Slavin  
```

Comment:
if you're comfortable with it, feel free to put yourself as primary author, @erslavin -- I feel you've done the most heavy lifting with this effort.

Reply:
Agreed!

Reply:
@erslavin -- sounds like in general NV is using the term AECO (architecture, engineering, and construction, and operations) rather than AEC-- let's reflect that here as well. Thanks!

Reply:
made the adjustment to AECO